src/create-handler-map.ts

```diff
@@ -26,7 +26,47 @@ export type CreateHandlerMap<TPrevState> = <
 >(
   actionCreators: TActionCreator | TActionCreator[],
   handler: Handler<TPrevState, TAction, TNextState>
-) => HandlerMap<TPrevState, TAction, TNextState>
+) => HandlerMap<TPrevState, TAction, TNextState>) & {
+  default: <
+    TActionCreator extends ActionCreator<any>,
+    TNextState extends TPrevState,
+    TAction extends AnyAction = TActionCreator extends (...args: any[]) => infer T
+      ? T
+      : never
+    >(
+    handler: Handler<TPrevState, TAction, TNextState>
+  ) => { default: Handler<TPrevState, TAction, TNextState> }
+}
```

Comment:
I think we should separate this huge type into `createCustomHandlerMap` and `createDefaultHandlerMap` types and union them into `createHandlerMap` type.

Reply:
These types have been divided. I preferred `createCustomHandlerMap` and `createDefaultHandler`, not a `create...Map` for the second one. Such type names just sound more meaningful. What do you think?